### ISSUE TITLE
Pedestal places depleted crystal in inventory not accepting them

### ISSUE THREAD

**Issue by Xandaros** [McJty/DeepResonance#104]
If you place a specialised inventory next to the pedestal and the pedestal collects the depleted crystal, it will place it in the inventory, even if the inventory does not normally accept depleted crystals.
(Or crystals in general)

**Comment by McJty** [comment]
Then I think that other inventory isn't implementing things right because DR is doing the needed checks. What block was this?

**Comment by Xandaros** [comment]
It was the adapter from open computers, I believe. The thing that lets you interface with non-computer blocks.

**Comment by McJty** [comment]
Then I suspect the error is in that block

**Comment by McJty** [comment]
What does a hopper do? That would be a way to test

**Comment by Xandaros** [comment]
Well, a hopper is meant to take crystals, so that doesn't really tell us anything.
The adapter, however, is only meant to take very specific items. (I think OC cards, but I'm not sure)

If you try to place any other item in there, like DR crystals, it won't let you. The pedestal, however, will happily place them in there.

**Comment by McJty** [comment]
No, I meant a hopper to the adapter. My guess is that the the hopper will insert 'invalid' items into the adapter too

**Comment by Xandaros** [comment]
Nope, the hopper does not insert the crystals into the adapter.
If you can't reproduce this, maybe you fixed it already?
I'm still using the same version that I used when reporting this (9e6c96813d21aa30bb0019839f61d1a06d34dfb3), which is a bit older.

**Comment by McJty** [comment]
Possibly although I didn't remember fixing something like this
